packages/button/src/css/index.js

```diff
@@ -144,6 +193,18 @@ export default {
     color: core.colors.gray03,
     background: core.colors.gray01
   },
+  [`.psds-button--disabled.psds-button--2020-colors`]: {
+    opacity: '50%',
+    cursor: 'not-allowed',
```

Comment:
The cursor is a new thing. So I think you're right. A for why It's necessary because we are no longer setting background on disabled buttons like [here](https://github.com/pluralsight/design-system/blob/48fa8dadb5b86b6064a49b20c3c1f6a137e5c9a0/packages/button/src/css/index.js#L134). Instead we're using an opacity approach.